src/cacao_zcap/mod.rs

```diff
@@ -0,0 +1,206 @@
+//! [CACAO-ZCAP](https://demo.didkit.dev/2022/cacao-zcap/) implementation
+
+mod core;
+pub use self::core::*;
+#[cfg(feature = "cacao-zcap")]
```

Comment:
You can remove this `#[cfg(...)]` statement as the entire module is already enabled for this feature.